carmin.yaml

```diff
@@ -10,514 +10,516 @@ info:
     name: CARMIN mailing list
     url: https://groups.google.com/d/forum/carmin
     email: carmin@googlegroups.com
-schemes:
-  - https 
-consumes:
-  - application/json
-produces:
-  - application/json
-
-securityDefinitions:
-  apiKey:
-      type: apiKey
-      name: apikey
-      in: header
-      description: An API key that must be provided by the platform.
 security: 
-  - apiKey: []
-
+- ApiKey: []
 paths:
   /platform:
     get:
+      summary: Returns information about the platform
```

Comment:
With this change (and other similar changes in this PR), the description field is not complete anymore. Is it correct ? It means that one can use the summary but to have a longer description, one must combine summary with description.

Reply:
The new version add this "summary" field that was not present before.
I don't think the description should also include the summary text, it seems redundant. But it's not clear in the OpenAPI spec.

Reply:
I have the feeling that strict redundancy is not the general rule. One can put a long complete text in description and a shorter one in summary. We are facing redundancy in CARMIN because we have very short descriptions that cannot be summarized.